src/renderer/services/synchronizer/network.js

```diff
@@ -0,0 +1,3 @@
+export default async synchronizer => {
+  await synchronizer.$store.dispatch('network/updateData')
```

Comment:
Do we need to update this data often?

Reply:
I think this is a yes and no answer. No, because of there being very little milestones. Yes, because when we are at a milestone we could do with knowing about it fairly promptly. The alternative is to advise that users need to reload the app for the wallet to acknowledge a milestone has been reached. Which do you think?

Reply:
Ah, got it. Yeh, since the only use will be this milestone, I think we should advise users to reload.

Reply:
I've removed the auto-sync @luciorubeens